src/tools/annotation/bidirectionalTool/utils/calculateLongestAndShortestDiameters.test.js

```diff
@@ -0,0 +1,119 @@
+import calculateLongestAndShortestDiameters from './calculateLongestAndShortestDiameters.js';
+import lodash from 'lodash';
+
+const measurementData = {
+  handles: {
+    start: {
+      x: 290.6229508196721,
+      y: 272.7868852459016,
+    },
+    end: {
+      x: 361.96721311475403,
+      y: 191.99999999999997,
+    },
+    perpendicularStart: {
+      x: 346.49180327868845,
+      y: 250.22950819672127,
+    },
+    perpendicularEnd: {
+      x: 306.0983606557376,
+      y: 214.55737704918027,
+    },
+  },
+};
+
+describe('CalculateLongestAndShortestDiameters.js', () => {
+  beforeEach(() => {
+    console.error = jest.fn();
+    console.error.mockClear();
+    console.warn = jest.fn();
+    console.warn.mockClear();
+  });
+
+  describe('calculate diameters', () => {
```

Comment:
I don't know that you need a second describe. Usually 1 per file is okay 👍

Reply:
I probably did that to separate the success scenarios with the failed (throw exceptions or logs) ones. but you are right, I removed it.